tests/functional/authentication/test_authentication.py

```diff
@@ -54,6 +54,10 @@ def test_privileged_access(self):
                                                 "invalid:invalid")
         self.assertIsNone(sessionToken, "Invalid credentials gave us a token!")
 
+        # A non-authenticated session should return an empty user
+        user = auth_client.getLoggedInUser()
+        self.assertEqual(user, "")
```

Comment:
If `return None` this test case should be updated. There is an `assertIsNone` method.